### ISSUE TITLE
ModuleNotFoundError with cupy 7.1.0 installed from conda-forge

### ISSUE THREAD

**Issue by shwina** [cupy/cupy#2971]
After installing cupy 7.1.0 from conda-forge, I get the following error, which is mitigated by downgrading to 7.0.0:

```
In [1]: import cupy as cp                                                                                                                                                                     

In [2]: a = cp.asarray([1, 2, 3])                                                                                                                                                             

In [3]: a.max()                                                                                                                                                                               
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-3-f432eed13108> in <module>
----> 1 a.max()

cupy/core/core.pyx in cupy.core.core.ndarray.max()

cupy/core/core.pyx in cupy.core.core.ndarray.max()

cupy/core/_routines_statistics.pyx in cupy.core._routines_statistics._ndarray_max()

cupy/cuda/cub.pyx in cupy.cuda.cub.cub_reduction()

ModuleNotFoundError: No module named 'cupy.core._reduction'
```

Thanks!

**Comment by jakirkham** [comment]
Which Python and CUDA versions are you using?

**Comment by shwina** [comment]
Python: 3.7.6
CUDA: 10.0

**Comment by jakirkham** [comment]
It appears there is [no `cupy.core._reduction` in v7.1.0]( https://github.com/cupy/cupy/tree/v7.1.0/cupy/core ). That was added in PR ( https://github.com/cupy/cupy/pull/2785 ), which only appears to be in `master`. Looks like backport PR ( https://github.com/cupy/cupy/pull/2848 ) caused the issue.

**Comment by jakirkham** [comment]
@emcastillo @grlee77, any thoughts on how we should address this?

**Comment by jakirkham** [comment]
@shwina, do you see this with CuPy 6.7.0 as well?

**Comment by leofang** [comment]
Please set `cp.cuda.cub_enabled=False` for the time being.

@kmaehashi @emcastillo We really need to get #2584 and #2598 merged asap...CUB bugs just get unnoticed by  CI once again...

**Comment by jakirkham** [comment]
I don't know if this is all that is needed or if there is more work needed, but submitted PR ( https://github.com/cupy/cupy/pull/2972 ), which fixes the `ModuleNotFoundError`.

**Comment by jakirkham** [comment]
> Please set `cp.cuda.cub_enabled=False` for the time being.

That workaround works for me. 😄

**Comment by grlee77** [comment]
> @shwina, do you see this with CuPy 6.7.0 as well?
> Edit: Have just tested CuPy 6.7.0 and do not see the issue. So it is only in 7.1.0.

Yes, I don't think there is any use of CUB in 6.x, so that branch should be okay.

**Comment by jakirkham** [comment]
FYI we patched this in the conda-forge package. ( https://github.com/conda-forge/cupy-feedstock/pull/32 )

**Comment by shwina** [comment]
This issue is resolved for me; please feel free to close it. Thanks everyone!

**Comment by leofang** [comment]
It'd be nice to have a hot-fix release v7.1.1 to address this, even though CUB is not enabled in the PyPI wheels yet.

**Comment by kmaehashi** [comment]
Thanks for reporting, we're planning v7.1.1 hotfix release tomorrow.
https://github.com/cupy/cupy/pull/2986

**Comment by kmaehashi** [comment]
> @kmaehashi @emcastillo We really need to get #2584 and #2598 merged asap...CUB bugs just get unnoticed by CI once again...

Sorry for the delay in these PRs. I will merge by the v7.2.0 release.

**Comment by jakirkham** [comment]
Thanks all for the help! 😄

**Comment by kmaehashi** [comment]
I've released the v7.1.1 hotfix.
https://github.com/cupy/cupy/releases/tag/v7.1.1

**Comment by leofang** [comment]
Thank you, @kmaehashi!
